mousestyles/behavior/tests/test_behavior.py

```diff
@@ -1,6 +1,55 @@
+"""Test behavior functions
+"""
+
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
 
 from mousestyles import behavior
+from mousestyles import data
+from mousestyles import intervals
+
+from collections import defaultdict
+
+
+def test_create_intervals():
+
+    # Checking intervals are being correctly loaded for a particular
+    # activity type and mouse-day
+    load_ints1 = behavior.create_intervals(
+        'F', strain=0, mouse=0, day=0)
+
+    # Calculate the sum of the loaded intervals
+    ints1_sum = sum(load_ints1)
+
+    # We now do the above calculations using a manual query
+    l_ints_manual = data.load_intervals('F')
+    l_ints_manual = l_ints_manual.query('strain == 0 and mouse == 0 and \
+                                         day == 0')[['start', 'stop']]
+    ints_manual = intervals.Intervals(l_ints_manual)
+    ints_manual_sum = sum(ints_manual)
+
+    assert ints1_sum.all() == ints_manual_sum.all()
```

Comment:
I don't think this really makes sense. Doesn't `all` test if all values in the array are truth-y? If that's true, this will almost always be `True == True` (unless there are zeros). I think it probably should be `(ints1_sum == ints_manual).all()`, or  equivalently, `np.all(ints1_sum == ints_manual)`.